src/styles/main.scss

```diff
@@ -0,0 +1,137 @@
+@import "./variables";
+@import "./reset";
+@import "./animation-stepOn60";
+@import "./extends";
+@import "./mixins";
+
+.page {
+  background: $c-bg-gradient;
+  height: 100vh;
+  margin: 0;
+
+  @extend %flex-center;
+}
+
+.watch {
+
+  & > * {
+    opacity: 0.68;
+  }
+
+  margin: auto;
+  width: $size-main;
+  height: $size-main;
+
+  position: relative;
+
+  @extend %flex-center;
+  align-items: center;
+
+  border-radius: 50%;
+  border-style: dotted;
+  border-color: $c-circles;
+
+  border-width: $size-circle;
+
+  &__pad {
+    opacity: 0.5;
+    position: absolute;
+
+    width: $size-pad;
+    height: $size-pad;
+
+    &-lines {
+      stroke: $c-lines;
+      stroke-width: 3px;
+    }
+  }
+
+  &::after {
+    content: "";
+
+    @extend %abs100;
+
+    border-radius: 50%;
+  }
+
+  &__seconds {
+
+    width: $size-second;
+    height: $size-second;
+
+    position: absolute;
+    z-index: 3;
+
+    &-second {
+
+      @extend %abs100;
+      @include rotation(60s);
+
+      &::before {
+        border-radius: $size-br-radius;
+
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 49%;
+        width: 2%;
+        height: 50%;
```

Comment:
It seems that all these groups of values can be calculated using one function or mixin. You've hardcoded a lot of data, it's bad practice.

Reply:
created 2 mixins for purpose to draw hands and their axis dots more mathematically effective